projects/default/controllers/sumo_supervisor/Objects.py

```diff
@@ -102,22 +102,26 @@ class Vehicle:
 
     def __init__(self, node):
         """Initialize and get the required fields from the vehicle node."""
+        if node.getTypeName() in ['Solid']:
+            self.node_car = node.getField('children').getMFNode(0)
```

Comment:
I would name this member field `car_node` instead of `node_car` which sounds french style.